ocaml/tests/alerts/test_expiry_alert.ml

```diff
@@ -0,0 +1,407 @@
+(*
+ * Copyright (C) Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+open Expiry_alert
+
+let date_of = Xapi_stdext_date.Date.of_string
+
+let host_server_cert_expiring_conditions =
+  [
+    (7, Api_messages.host_server_certificate_expiring_07)
+  ; (14, Api_messages.host_server_certificate_expiring_14)
+  ; (30, Api_messages.host_server_certificate_expiring_30)
+  ]
```

Comment:
These belong to the tests for the certificate checks, not to the library tests

Reply:
Create a set of test message ids to use for testing, now no message_id from Api_messages is used in the UT.